src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -932,25 +947,25 @@ private NuGetProject CreateNuGetProject(Project envDTEProject, INuGetProjectCont
         }
 
         // REVIEW: This might be inefficient, see what we can do with caching projects until references change
-        internal static IEnumerable<EnvDTE.Project> GetDependentEnvDTEProjects(IDictionary<string, List<EnvDTE.Project>> dependentEnvDTEProjectsDictionary, EnvDTE.Project envDTEProject)
+        internal static IEnumerable<IVsProjectAdapter> GetDependentProjects(IDictionary<string, List<IVsProjectAdapter>> dependentProjectsDictionary, IVsProjectAdapter vsProjectAdapter)
         {
             Debug.Assert(ThreadHelper.CheckAccess());
```

Comment:
Correct, it's hard to change every occurrence. I might've missed hundreds of them :(